db/migrate/20180710052501_adding_footer_content_to_db.rb

```diff
@@ -0,0 +1,17 @@
+class AddingFooterContentToDb < ActiveRecord::Migration[5.2]
+  def change
+    str = <<-EOF
+    <p>
+       Powered by
+       <a href="https://github.com/NYPL/transcript-editor">Open Transcript Editor</a>
+       which was developed by the
+       <a href="http://www.nypl.org">New York Public Library</a>
```

Comment:
I think this should be a `target="_blank"` as it's a new website

Reply:
@Rhiana , I think we can change it with the new design. I've extracted it as same as in current layout :)